c2corg_common/attributes.py

```diff
@@ -612,7 +612,8 @@
 
 article_categories = [
     'mountain_environment',
-    'gear_and_technique',
+    'gear',
+    'technical',
```

Comment:
The string 'gear' exist as outing field, 'technical' doesn't exist.
In V5, it's 'gear_article' and 'technique_article'.
But as these strings are used in the url of article search, I would use shorter strings as the V5 strings.
Ex : filter on 4 categories : 
- V5 : http://www.camptocamp.org/articles/list/ccat/1-11-4-7
- V6 : http://www.demov6.camptocamp.org/articles#acat=mountain_environment%252Cgear_and_technique%252Ctopoguide_supplements%252Csoft_mobility